extensions/wikia/Forum/ForumBoard.class.php

```diff
@@ -100,7 +102,7 @@ private function getBoardInfoFromMaster() {
 		$forumBoardInfo->setName( $this->getTitle()->getText() );
 		$forumBoardInfo->setUrl( $this->getTitle()->getFullURL() );
 		$forumBoardInfo->setDescription( $this->getDescriptionWithoutTemplates() );
-		$memKey = wfMemcKey( 'forum_board_info', $this->getId() );
+		$memKey = wfMemcKey( self::FORUM_BOARD_INFO, self::CACHE_VERSION , $this->getId() );
```

Comment:
A helper method is still better. Here you can still omit one instance of memcache key generation.